navapbc/wic-participant-recertification-portal-demo
# Create static demo version of the staff portal

## Ticket

N/A

## Changes
> What was added, updated, or removed in this PR.

- Remove authentication
- Remove login page
- Add static demo content
- Add static download files
- Clean up docker compose
- Remove database connection

## Context for reviewers
> Testing instructions, background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers. Explain how the changes were verified.

## Testing
> Screenshots, GIF demos, code examples or output to help show the changes working as expected. ProTip: you can drag and drop or paste images into this textbox.

1. Checkout this feature branch
2. `docker compose up -d --build`
3. Go to `localhost:3033` (dev build) and verify the page loads and the document links work
4. Go to `localhost:3333` (prod build) and verify the page loads and the document links work
5. `docker compose down`

Should look like this:

![Screenshot 2023-07-05 at 10-14-26 Montana WIC Staff Portal](https://github.com/navapbc/wic-participant-recertification-portal-demo/assets/67701/b210d95c-8896-4b0c-8b42-e9a6c509b846)
